### PR TITLE
chore(vulnfeeds): pin to Cloud SDK 485.0.0

### PR DESCRIPTION
**Pull request by andrewpollock**
Harmonise explicitly on 485.0.0 until GCS performance regression is resolved

### CODE REVIEW

File: go.work
Comment (hogo6002) [inline go.work:1]: I'm curious why would we have this `go.work` file at the first place if it does nothing.

**Review by hogo6002 — APPROVED** [review]
LGTM!

File: go.work
Comment (andrewpollock) [reply go.work:1]: https://www.reddit.com/r/golang/comments/15v8i06/why_gowork_file_should_not_be_committed/ talks about it in more detail

Comment (another-rex) [reply go.work:1]: Do you want to add go.work file to the gitignore?

**Pull request merged**